spec/index.bs

```diff
@@ -7402,7 +7502,13 @@ dictionary GPURenderPipelineDescriptor : GPUPipelineDescriptorBase {
             <div data-timeline=content>
                 [=Content timeline=] steps:
 
-                1. [=Resolve=] |promise| with |pipeline|.
+                1. If |pipeline| is...
+                    <dl class=switch>
+                        : [=valid=]
+                        :: [=Resolve=] |promise| with |pipeline|.
+                        : [=invalid=]
```

Comment:
Don't we need to handle internal and validation errors separately here like in the compute pipeline algorithm?